src/translations/et.yml

```diff
@@ -0,0 +1,34 @@
+consentModal:
+  title: Isikuandmete kogumine
+  description: >
+    Siit saate vaadata ja hallata teavet, mida me teie kohta kogume.
+  privacyPolicy:
+    name: privaatsustingimused
+    text: >
+      Lisateabe saamiseks lugege palun meie {privacyPolicy}
+consentNotice:
+  changeDescription: Pärast teie viimast külastust on toimunud muudatusi, palun uuendage oma nõusolekut.
+  description: >
+    Me kogume ja töötleme teie isikuandmeid järgmistel eesmärkidel: {purposes}.
+  learnMore: Lisateave
+accept: Nõustu
+acceptTitle: Nõustu küpsistega
+acceptAll: Nõustu kõigi rakendustega
+save: Salvesta
+saveData: Salvesta kogutud teabe seadistused
+decline: Keeldu
+declineAll: Keela kõik rakendused
+close: Sulge
+enabled: Lubatud
+disabled: Välja lülitatud
+app:
+  optOut:
+    title: (optOut)
```

Comment:
As with the german translation, I think this could be improved a bit.
Do you know of any estonian word that could fit?
https://github.com/empreinte-digitale/orejime/pull/95#discussion_r1224096536

Reply:
Updated for `Opt-Out`, cf https://github.com/empreinte-digitale/orejime/pull/95#discussion_r1295906222